Automattic/wp-calypso
# Add/reader lists/list delete

#### Changes proposed in this Pull Request

* Hooks up deleting a reader list

#### Testing instructions



* Find a list you don't like
* Delete it
* It should delete the list and redraw the screen indicating the list was deleted
